src/Controller/ControllerManager.php

```diff
@@ -73,10 +74,10 @@ public function injectControllerDependencies($controller, ServiceLocatorInterfac
             // is why the shared EM injection needs to happen; the conditional
             // will always pass.
             $events = $controller->getEventManager();
```

Comment:
Will need to check this against an interface (as per comment below), as the *aware* interface defines no getter :-(